espnet/nets/pytorch_backend/frontend_asr.py

```diff
@@ -0,0 +1,212 @@
+                            help='')
+        parser.add_argument('--wdropout-rate', type=float, default=0.0,
+                            help='')
+        parser.add_argument('--wpe-taps', type=int, default=5,
+                            help='')
+        parser.add_argument('--wpe-delay', type=int, default=3,
+                            help='')
+        parser.add_argument('--use-dnn-mask-for-wpe', type=strtobool,
+                            default=False,
+                            help='Use DNN to estimate the power spectrogram. '
+                                 'This option is experimental.')
+        # Beamformer related
+        parser.add_argument('--use-beamformer', type=strtobool,
+                            default=True, help='')
+        parser.add_argument('--btype', default='blstmp', type=str,
+                            choices=['lstm', 'blstm', 'lstmp', 'blstmp', 'vgglstmp', 'vggblstmp', 'vgglstm', 'vggblstm',
+                                     'gru', 'bgru', 'grup', 'bgrup', 'vgggrup', 'vggbgrup', 'vgggru', 'vggbgru'],
+                            help='Type of encoder network architecture '
+                                 'of the mask estimator for Beamformer.')
+        parser.add_argument('--blayers', type=int, default=2,
+                            help='')
+        parser.add_argument('--bunits', type=int, default=300,
+                            help='')
+        parser.add_argument('--bprojs', type=int, default=300,
+                            help='')
+        parser.add_argument('--badim', type=int, default=320,
+                            help='')
+        parser.add_argument('--ref-channel', type=int, default=-1,
+                            help='The reference channel used for beamformer. '
+                                 'By default, the channel is estimated by DNN.')
+        parser.add_argument('--bdropout-rate', type=float, default=0.0,
+                            help='')
+        # Feature transform: Normalization
+        parser.add_argument('--stats-file', type=str, default=None,
+                            help='The stats file for the feature normalization')
+        parser.add_argument('--apply-uttmvn', type=strtobool, default=True,
+                            help='Apply utterance level mean '
+                                 'variance normalization.')
+        parser.add_argument('--uttmvn-norm-means', type=strtobool,
+                            default=True, help='')
+        parser.add_argument('--uttmvn-norm-vars', type=strtobool, default=False,
+                            help='')
+        # Feature transform: Fbank
+        parser.add_argument('--fbank-fs', type=int, default=16000,
+                            help='The sample frequency used for '
+                                 'the mel-fbank creation.')
+        parser.add_argument('--n-mels', type=int, default=80,
+                            help='The number of mel-frequency bins.')
```

Comment:
Mel

Reply:
considering `--fbank-xxx` options, `--fbank-dim` is more clear

Reply:
Mel?